diff --git a/node_modules/@deldev/capacitor-google-auth/android/build.gradle b/node_modules/@deldev/capacitor-google-auth/android/build.gradle
index e02a3b7..2f39212 100644
--- a/node_modules/@deldev/capacitor-google-auth/android/build.gradle
+++ b/node_modules/@deldev/capacitor-google-auth/android/build.gradle
@@ -8,7 +8,7 @@ ext {
 
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
         google()
     }
     dependencies {
@@ -46,7 +46,6 @@ android {
 
 repositories {
     google()
-    jcenter()
     mavenCentral()
 }
 
